docs/documentation/components/application-base.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="nl">
```

Reply:
Ah yes, great find. I will solve this for all instances in a separate pr.